### PR TITLE
Fix the PAL not showing user info for user who opened PAL

### PR DESCRIPTION
**Pull request by zfox23**
There's a problem in Master where the PAL will frequently be very broken - the DisplayName field will sometimes be blank, the user's profile picture will not show, the user's admin status will not be reflected properly, etc. This PR fixes that.

Currently DNM because it's 5PM on Cinco de Mayo, it's a Friday, everyone else is drinking margaritas, and I want to make sure this is a valid solution.

**Test Plan:**
1. Log in.
2. Go into a domain in which you're an admin (your sandbox works).
3. Open the PAL. Verify that the "Admin" options appear for you. Verify that your profile picture, username, and display name show up in the top left of the PAL.
4. Go into a domain in which you're not an admin (like dev-welcome, probably).
5. Open the PAL. Verify that your profile picture, username, and display name show up in the top left of the PAL.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: fdae4a2aa0f7fc8738d5394a5ca89069b9214c00 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-fdae4a2.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-fdae4a2.dmg

**Comment by howard-stearns** [comment]
See https://highfidelity.fogbugz.com/f/cases/4641/ignoring-late-avatar-identity-is-throwing-away-data

File: libraries/avatars/src/AvatarData.cpp [hunk 1]
```diff
@@ -1497,7 +1497,7 @@ QUrl AvatarData::cannonicalSkeletonModelURL(const QUrl& emptyURL) const {
 
 void AvatarData::processAvatarIdentity(const Identity& identity, bool& identityChanged, bool& displayNameChanged) {
 
-    if (identity.updatedAt < _identityUpdatedAt) {
+    if (identity.updatedAt < _identityUpdatedAt && !(DependencyManager::get<NodeList>()->getRequestsDomainListData())) {
```
Comment (ZappoMan) [inline libraries/avatars/src/AvatarData.cpp]: these seems wrong.

What your new logic is doing is going to over-write newer identity packets with older data in the event that the PAL is open... that doesn't seem correct.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e2c40daf357252c8278c70471df1c655f25106c0 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-e2c40da.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-e2c40da.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d37e8ba7542555dea19c0d1979b7ca4159fbe742 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-d37e8ba.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-d37e8ba.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d7e4f022911afa2460f8a164ba5859716caf56d9 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-d7e4f02.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-d7e4f02.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5cb4351ca088e2d90a75edb47c6030dc1de8b372 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-5cb4351.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-5cb4351.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ae983658becf18702d6cd957016f84bda0ab3141 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-ae98365.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-ae98365.dmg

**Comment by zfox23** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: ae983658becf18702d6cd957016f84bda0ab3141 available at: https://gist.github.com/f7f744265aa3ef650aea5f16c75500c8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-ae98365.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-ae98365.dmg

**Comment by howard-stearns** [comment]
Looks good!

**Comment by zfox23** [comment]
test this please

**Comment by zfox23** [comment]
Windows installer EXE: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10398-15574f5.exe

**Comment by CainFoool** [comment]
Testing

**Comment by CainFoool** [comment]
Testing Complete. Profile Picture, Username and Display Name all appeared in both Sandbox and dev-welcome. Admin tools appeared in Sandbox only.

**Pull request merged**